Bottom: 540e4b48729f21e0955382d5851967004ef32105
Top:    cd777575545a8e99abcfdb43a3a37d2eb8adbe87
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 12:25:44 +0800

Refresh of autofs-5.0.7-fix-portmap-version.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 56d083a..f8b5d19 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -78,6 +78,7 @@
 - fix bad mkdir permission on create.
 - setup program map env from macro table.
 - add short host name standard marco variable.
+- fix portmap version.
 
 25/07/2012 autofs-5.0.7
 =======================
